static/app/components/nav/orgDropdown.tsx

```diff
@@ -0,0 +1,287 @@
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
+import {useUser} from 'sentry/utils/useUser';
+
+function createOrganizationMenuItem(): MenuItemProps {
+  const configFeatures = ConfigStore.get('features');
+  const sentryUrl = localizeDomain(ConfigStore.get('links').sentryUrl);
+  const route = '/organizations/new/';
+  const canCreateOrg = ConfigStore.get('features').has('organizations:create');
+
+  const menuItemProps: MenuItemProps = {
+    key: 'create-organization',
+    label: (
+      <CreateOrganizationMenuItem>
+        <IconAdd />
+        {t('Create a new organization')}
+      </CreateOrganizationMenuItem>
+    ),
+  };
+
+  if (configFeatures.has('system:multi-region')) {
+    menuItemProps.externalHref = sentryUrl + route;
+  } else {
+    menuItemProps.to = route;
+  }
+
+  return {
+    key: 'create-organization-section',
+    children: [menuItemProps],
+    hidden: !canCreateOrg,
+  };
+}
+
+function MenuOrgSummary() {
+  const organization = useOrganization();
+  const {projects} = useProjects();
+
+  return (
+    <MenuOrgSummaryWrapper>
+      <OrganizationAvatar organization={organization} size={36} />
+      <div>
+        <MenuOrgSummaryName
+          pendingDeletion={organization.status.id === 'pending_deletion'}
+        >
+          {organization.name}
+        </MenuOrgSummaryName>
+        <ProjectCount>{tn('%s Project', '%s Projects', projects.length)}</ProjectCount>
+      </div>
+    </MenuOrgSummaryWrapper>
+  );
+}
+
+function SwitchOrganizationItem({organization}: {organization: OrganizationSummary}) {
+  return (
+    <SwitchOrganizationWrapper>
+      {organization.status.id === 'pending_deletion' ? (
+        <PendingDeletionAvatar data-test-id="pending-deletion-icon">
+          <IconWarning size="xs" color="gray200" />
+        </PendingDeletionAvatar>
+      ) : (
+        <OrganizationAvatar organization={organization} size={22} />
+      )}
+      <SwitchOrganizationItemName
+        pendingDeletion={organization.status.id === 'pending_deletion'}
+      >
+        {organization.name}
+      </SwitchOrganizationItemName>
+    </SwitchOrganizationWrapper>
+  );
+}
+
+export function OrgDropdown() {
+  const api = useApi();
+
+  const config = useLegacyStore(ConfigStore);
+  const organization = useOrganization();
+  const user = useUser();
+
+  // It's possible we do not have an org in context (e.g. RouteNotFound)
+  // Otherwise, we should have the full org
+  const hasOrgRead = organization?.access?.includes('org:read');
+  const hasMemberRead = organization?.access?.includes('member:read');
+  const hasTeamRead = organization?.access?.includes('team:read');
+
+  const {organizations} = useLegacyStore(OrganizationsStore);
+
+  function handleLogout() {
+    logout(api);
+  }
+
+  return (
+    <DropdownMenu
+      trigger={props => (
+        <OrgDropdownTrigger
```

Comment:
Do you need to style `OrgDropdownTrigger`, or can you just give `AvatarWrapper` a size?

Reply:
I want a larger click area than the default border size="zero" allows for and I also want it to perfectly match the size of the items below it, so I think this is the best way to achieve that. I did find that AvatarWrapper was unnecessary so I removed that